spack-ext/repos/spack-stack/packages/ufs-utils-env/package.py

```diff
@@ -30,7 +30,7 @@ class UfsUtilsEnv(BundlePackage):
     depends_on("w3emc")
     depends_on("sigio")
     depends_on("sfcio")
-    depends_on("zlib")
```

Comment:
I am fine with this change, but `depends_on("zlib-api" didn't work)?

Reply:
That's odd, I thought I changed it to zlib-api...